src/jace/translator/sub_translators/__init__.py

```diff
@@ -0,0 +1,92 @@
+# JaCe - JAX Just-In-Time compilation using DaCe (Data Centric Parallel Programming)
+#
+# Copyright (c) 2024, ETH Zurich
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Module collecting all built-in subtranslators."""
```

Comment:
Why do we need to distinguish between primitive translator based on who implemented them? Why do we need to delete primitive translators? I don't see the use cases, and certainly not now, so to me this module looks like it's bloated with functionality which is not currently needed.

Reply:
Distinguish them would allow that an external user can override a built in primitive.
At initialization, first the externally supplied translators would be created.
Then one would go through the build in, if a subtranslator was already registered for it we know that it came from the outside and was supposed to override the built in and we would skip it.
I think it is quite an elegant solution.